build/config/compiler/BUILD.gn

```diff
@@ -416,13 +416,15 @@ config("compiler") {
       "-funwind-tables",
       "-fno-short-enums",
       "-nostdinc++",
-      # TODO (https://github.com/flutter/flutter/issues/75348).
-      "-mno-outline-atomics",
     ]
     if (!is_clang) {
       # Clang doesn't support these flags.
       cflags += [ "-finline-limit=64" ]
     }
+    if (current_cpu == "arm64") {
```

Comment:
shoudln't this be `target_cpu`?

Reply:
`current_cpu` is generally preferred in build rules and will be equivalent to `target_cpu` in this case

See https://chromium.googlesource.com/experimental/chromium/src/+/refs/wip/bajones/webvr/tools/gn/docs/cross_compiles.md#as-a-build_gn-author

Reply:
`target_cpu` will also hold on host builds. `current_cpu` is accurate.

Reply:
Ah, good to know :-) LGTM!